From c611d3d31ab99d9a13f192294b2eb393d3540fa6 Mon Sep 17 00:00:00 2001
From: Geert Uytterhoeven <geert@linux-m68k.org>
Date: Tue, 1 Jun 2021 15:57:52 +0200
Subject: [PATCH 05/42] [WIP] dt-bindings: clock: starfive: Add preliminary
 JH7100 Clock Definitions

Add all clock outputs for the StarFive JH7100 Clock Generator, based on
the list of fixed-frequency clocks defined in jh7100.dtsi.

To be verified against documentation when it becomes available.

Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
---
 .../clock/starfive-jh7100-clkgen.h            | 29 +++++++++++++++++++
 1 file changed, 29 insertions(+)
 create mode 100644 include/dt-bindings/clock/starfive-jh7100-clkgen.h

diff --git a/include/dt-bindings/clock/starfive-jh7100-clkgen.h b/include/dt-bindings/clock/starfive-jh7100-clkgen.h
new file mode 100644
index 000000000000..5f6807c82ec5
--- /dev/null
+++ b/include/dt-bindings/clock/starfive-jh7100-clkgen.h
@@ -0,0 +1,29 @@
+/* SPDX-License-Identifier: (GPL-2.0 OR MIT) */
+/*
+ * Copyright (C) 2021 Glider bv
+ */
+#ifndef __DT_BINDINGS_CLOCK_STARFIVE_JH7100_CLOCK_H__
+#define __DT_BINDINGS_CLOCK_STARFIVE_JH7100_CLOCK_H__
+
+/* StarFive JH7100 clocks FIXME PRELIMINARY */
+#define JH7100_CLK_AXI		0
+#define JH7100_CLK_AHB0		1
+#define JH7100_CLK_AHB2		2
+#define JH7100_CLK_APB1		3
+#define JH7100_CLK_APB2		4
+#define JH7100_CLK_VPU		5
+#define JH7100_CLK_JPU		6
+#define JH7100_CLK_PWM		7
+#define JH7100_CLK_DWMMC_BIU	8
+#define JH7100_CLK_DWMMC_CIU	9
+#define JH7100_CLK_UART		10
+#define JH7100_CLK_HS_UART	11
+#define JH7100_CLK_I2C0		12
+#define JH7100_CLK_I2C2		13
+#define JH7100_CLK_QSPI		14
+#define JH7100_CLK_SPI		15
+#define JH7100_CLK_GMAC		16
+#define JH7100_CLK_HF		17
+#define JH7100_CLK_RTC		18
+
+#endif /* __DT_BINDINGS_CLOCK_STARFIVE_JH7100_CLOCK_H__ */
-- 
2.31.1

